### PR TITLE
REL: v1.3.0

### PR DESCRIPTION
**Pull request by franklin-feingold**
This PR marks the start of our 1 week [release process](https://github.com/bids-standard/bids-specification/blob/master/Release_Protocol.md#4-open-a-pull-request-against-the-master-branch). 

We are processing a minor release due to the impending addition of [Genetic Information](https://github.com/bids-standard/bids-specification/pull/395) into the Specification. 

This release also includes:
- Adds PDF rendering of the Specification
- Adds an Optional EthicsApprovals field
- Adds issue templates for GitHub
- Adds GitHub maintainer account to handle automatic work

### CODE REVIEW

**Comment by sappelhoff** [comment]
When will we merge https://github.com/bids-standard/bids-specification/pull/395 (genetic information)?

**Comment by franklin-feingold** [comment]
let's say at the end of the week (pending comments)? sharing on our channels (twitter and the google group)

**Comment by effigies** [comment]
I thought genetics already had plenty of opportunity for comment. But I'm also not in a huge rush.

**Comment by CPernet** [comment]
must have been the BEP wit the most calls ever

**Comment by nicholst** [comment]
I want to again call out and congratulate @Arshitha for his herculean efforts with the PDF rendering #431 (with key support, as always, from @sappelhoff & @franklin-feingold).  From #436 I just took a quick look at the current [PDF rendering](https://circleci.com/api/v1.1/project/github/bids-standard/bids-specification/latest/artifacts/0/bids-spec.pdf?branch=master), and was struck immediately with the text from `src/index.md`, on the first content page:

> This site serves as an online resource to see the current state of the Brain Imaging Data Structure (BIDS) specification.

While I realise the goal was "Render the website as a PDF", with the exception of this "You're reading a website" prose, I think the document is really polished and stands on its own as a professional, published standards document.  

This leads me to wonder:   Should (1) the PDF rendering be changed to omit `src/index.md`, as material specific to the web-rendering of BIDS, or (2) should the text of `src/index.md` be revised to read more platform-independent.  I note that, from a quick scan, I don't see text/tone in the remainder of the text from website/PDF that is so platform specific.

I'm happy to make  PR but I'm not sure which direction to go in.  (We could also have a `src/index-pdf.md`, which is a PDF-rendering specific preface that parallels `index.md`) (Or, we resolutely want to remind people BIDS is a website-based standard, and leave it as.)

**Comment by sappelhoff** [comment]
I like your suggestion (2) @nicholst --> should the text of src/index.md be revised to read more platform-independent.

**Comment by nicholst** [comment]
> I like your suggestion (2) @nicholst --> should the text of src/index.md be revised to read more platform-independent.

OK, FWIW, see a trivial PR #437 that changes just the first sentence of `index.md`.

As an aside, "Chapter 1" really reads like a Preface.  @Arshitha is it as easy as changing `index.md` from
```
# The Brain Imaging Data Structure
```
to
```
# The Brain Imaging Data Structure {epub:type=preface}
```
in order to get it called a Preface, and then "Chapter 2: Introduction" becomes "Chapter 1".  (see https://pandoc.org/MANUAL.html#the-epubtype-attribute)

**Comment by chrisgorgo** [comment]
Just to clarify - is the intention to release 1.3.0 before or after BEP 018 (Genetic information) is merged?

**Comment by franklin-feingold** [comment]
the intention is to release v1.3.0 after BEP 018 (Genetic Information) is merged

**Comment by CPernet** [comment]
waiting on @rwblair @effigies to help me solve the last bit in [395](https://github.com/bids-standard/bids-specification/pull/395#issuecomment-602741736) because I'm lost and don't know where to make the change :-( (+validator update)

**Comment by rwblair** [comment]
@CPernet I'm a bit lost in the conversation for #395 and what needs to be done. To confirm I just need to update the validator to allow "GeneticLevel" in genetic_info.json to take an array of strings instead of just a single string. I will also update the examples to reflect this.

Are there any other changes that I should make?

**Comment by CPernet** [comment]
@rwblair yes that's all that is needed for the validator
I am also lost as to what has been merged - I need to add the definitions discussed above in the spec (but not sure where/which file at this stage)

**Comment by franklin-feingold** [comment]
@CPernet regarding the location of where the files are at - there is a [bep018](https://github.com/bids-standard/bids-specification/tree/bep018) branch that captures your extension.

The extension file is at: https://github.com/bids-standard/bids-specification/blob/bep018/src/04-modality-specific-files/08-genetic-descriptor.md

**Comment by CPernet** [comment]
great thx I've updated the descriptor

**Comment by Arshitha** [comment]
@nicholst Thank you for all the feedback on #431 ! 

I tried using the epub tag, initially, to generate the pdf but with little success. So I'm not sure if that'd fix it but I'll look into it.

**Comment by franklin-feingold** [comment]
Today https://github.com/bids-standard/bids-specification/pull/395 was merged into the standard! (woo @CPernet) The pdf is looking really good (cc: @nicholst @Arshitha) 

I think we are ready for this to be signed off and merged (after squeezing in: https://github.com/bids-standard/bids-specification/pull/442).  If so, this coming Monday (4/13) I'll cut the release
cc: @sappelhoff @effigies

**Comment by sappelhoff** [comment]
>  If so, this coming Monday (4/13) I'll cut the release

Yes :+1: , I just rebased this branch. Everything is ready for the release on Easter Monday :egg:

**Comment by nicholst** [comment]
Just a quick query... what's the link to the PDF associated with this PR? I used htis before but it doesn't seem to be associated with this PR: https://circleci.com/api/v1.1/project/github/bids-standard/bids-specification/latest/artifacts/0/bids-spec.pdf?branch=master ?

**Comment by sappelhoff** [comment]
@nicholst the link you posted works for the "dev" version of the PDF for the `master` branch, see the last part of the link syntax. In theory, you can change the `?branch=master` part to `?branch=rel/1.3.0`, but at that point, you might as well scroll down to the GitHub rendering of the checks and click manually.

I am talking about these badges and the "details" link they have:

image


---

USUALLY there would be one badge saying "build_docs_pdf" which would link you to the artifacts via the CircleCI interface (after some clicking) ... HOWEVER, that "build_docs_pdf" job runs only if all other jobs exit cleanly, see:

https://github.com/bids-standard/bids-specification/blob/7cd7c635343c822a9559ebb88e43a6d6adfa2bad/.circleci/config.yml#L182-L190

the linkchecker fails for this branch ... and it'll always fail for release PRs, because we are referencing a page that does not exist yet (see https://github.com/bids-standard/bids-specification/pull/435#pullrequestreview-388500370)

long story short: no PDF for this branch, but given that the DIFF of the release PRs are very minimal in general (see the release protocol), it may suffice to look at the `master` PDF.

**Comment by nicholst** [comment]
OK @sappelhoff!  Thanks for explaining.

File: src/CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [Unreleased](https://github.com/bids-standard/bids-specification/tree/HEAD)
+## [v1.3.0](https://bids-specification.readthedocs.io/en/v1.3.0/) (2020-4-14)
```
Comment (effigies) [inline src/CHANGES.md:3]:
```suggestion
## [v1.3.0](https://bids-specification.readthedocs.io/en/v1.3.0/) (2020-04-14)
```
